Corgibytes.Freshli.Cli/Functionality/Git/HistoryIntervalStop.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Corgibytes.Freshli.Cli.Functionality.Git;
+
+public class HistoryIntervalStop
+{
+    public HistoryIntervalStop(DateTimeOffset committedAt, string gitCommitIdentifier)
```

Comment:
I don't know if this matters, but this is semantically the opposite as the constructor for `GitCommit`: there, the parameters are "identifier, date", and here they're "date, identifier". I recommend picking one convention (identifier or date first?) and sticking to it to reduce guesswork.